diff --git a/bootable/recovery/Android.bp b/bootable/recovery/Android.bp
index 4728f2c..52267ec 100644
--- a/bootable/recovery/Android.bp
+++ b/bootable/recovery/Android.bp
@@ -51,21 +51,6 @@ cc_defaults {
     cpp_std: "c++17",
 }
 
-bootstrap_go_package {
-    name: "soong-libaosprecovery_defaults",
-    pkgPath: "bootable/recovery/libaosprecovery",
-    deps: [
-        "soong",
-        "soong-android",
-        "soong-cc"
-    ],
-    srcs: [
-        "libaosprecovery_defaults.go",
-        "soong/makevars.go"
-    ],
-    pluginFor: ["soong_build"]
-}
-
 libaosprecovery_defaults {
     name: "libaosprecovery_defaults"
 }
diff --git a/bootable/recovery/gui/Android.bp b/bootable/recovery/gui/Android.bp
index 6e8c7fb..f3b1e30 100644
--- a/bootable/recovery/gui/Android.bp
+++ b/bootable/recovery/gui/Android.bp
@@ -1,20 +1,3 @@
-bootstrap_go_package {
-    name: "soong-libguitwrp_defaults",
-    pkgPath: "bootable/recovery/gui",
-    deps: [
-        "soong",
-        "soong-android",
-        "soong-cc"
-    ],
-    srcs: [
-        "libguitwrp_defaults.go",
-        "../soong/copy.go",
-        "../soong/makevars.go",
-        "../orangefox_defaults.go"
-    ],
-    pluginFor: ["soong_build"]
-}
-
 libguitwrp_defaults {
     name: "libguitwrp_defaults"
 }
diff --git a/bootable/recovery/minuitwrp/Android.bp b/bootable/recovery/minuitwrp/Android.bp
index 7a56103..1efa469 100644
--- a/bootable/recovery/minuitwrp/Android.bp
+++ b/bootable/recovery/minuitwrp/Android.bp
@@ -1,18 +1,3 @@
-bootstrap_go_package {
-    name: "soong-libminuitwrp_defaults",
-    pkgPath: "bootable/recovery/minuitwrp",
-    deps: [
-        "soong",
-        "soong-android",
-        "soong-cc"
-    ],
-    srcs: [
-        "libminuitwrp_defaults.go",
-        "../soong/makevars.go"
-    ],
-    pluginFor: ["soong_build"]
-}
-
 libminuitwrp_defaults {
     name: "libminuitwrp_defaults"
 }
